mor/pyfeelpp-mor/tests/CMakeLists.txt

```diff
@@ -1,22 +1,22 @@
 ###  CMakeLists.txt; coding: utf-8 --- 
 
 
-set(PYFILES test_mor.py)
+set(PYFILES conftest.py test_mor.py)
 install(FILES ${PYFILES} DESTINATION ${FEELPP_PYTHON_MODULE_PATH}/feelpp)
 
 foreach(test mor )
   add_test (NAME feelpp_mor_python-tests-${test}
     COMMAND ${CMAKE_COMMAND} -E env
           LD_LIBRARY_PATH=${CMAKE_INSTALL_PREFIX}/${CMAKE_INSTALL_LIBDIR}:$ENV{LD_LIBRARY_PATH}
           PYTHONPATH=${CMAKE_INSTALL_PREFIX}/${FEELPP_PYTHON_MODULE_PATH}
-          ${PYTHON_EXECUTABLE} -m pytest -k ${test} -s ${CMAKE_CURRENT_SOURCE_DIR}
+          ${PYTHON_EXECUTABLE} -m pytest test_${test}.py -s -vv
```

Comment:
why do that ? it was more general before ?

Reply:
this is from pyrb #1787

Reply:
it was not working the way it was before, tests were not launched at all... But maybe that can be changed